thirdeye-spi/src/main/java/ai/startree/thirdeye/spi/util/MetricsUtils.java

```diff
@@ -15,13 +15,18 @@
 
 import io.micrometer.core.instrument.Metrics;
 import io.micrometer.core.instrument.Timer;
+import java.util.Optional;
 import java.util.concurrent.Callable;
+import org.checkerframework.checker.nullness.qual.NonNull;
+import org.checkerframework.checker.nullness.qual.Nullable;
 
 /**
  * Utilities for micrometer metrics.
  */
 public class MetricsUtils {
 
+  public final static String NAMESPACE_TAG = "namespace";
```

Comment:
this label will conflict with the kubernetes namespace label that may be injected by the cloud team 

I'd suggest: 
`thirdeye_workspace` to avoid potential collisions